### ISSUE TITLE
0.1 Scaling factor up() and down()

### ISSUE THREAD

**Issue by thwjoy** [pclucas14/iaf-vae#3]
Hi There

Nice work with this! I just have a quick question, there is a scaling factor of 0.1 at the following places in layers.py:

96:           return input + 0.1 * h
145:         return input + 0.1 * h, kl, kl_obj 

Can you tell me why the value of 0.1 has been introduced? Is it a hyperparameter?

**Comment by pclucas14** [comment]
Hi, 

that's a good question, I had the same one when I was coding up the paper. This was taken from [the official tensorflow code of the authors ](https://github.com/openai/iaf/blob/master/tf_train.py#L44). My best guess is that early in training this ensures a good gradient flow across all layers by favoring the residual connection. 

Hope this helps, 
Lucas
